File: Data Structures & Algorithms/daily-temperatures/submission-0.java
class Solution {
    public int[] dailyTemperatures(int[] temperatures) {
        int[] ans = new int[temperatures.length];
        for(int i = temperatures.length-1; i >= 0; i--){
            for(int j = i - 1; j >= 0; j--){
                if(temperatures[i] > temperatures[j])
                    ans[j] = i - j;
            }
        }
        return ans;
    }
}

File: Data Structures & Algorithms/k-closest-points-to-origin/submission-0.py
class Solution:
    def kClosest(self, points: List[List[int]], k: int) -> List[List[int]]:
        distance = []
        for point in points:
            dis = math.sqrt((point[0]) **2 + (point[1]) ** 2)
            heapq.heappush(distance, (dis, point))
        ans = []
        for i in range(k):
            ans.append(heapq.heappop(distance)[1])

        return ans    